src/state/tooltipSlice.ts

```diff
@@ -182,6 +194,12 @@ export const initialState: TooltipState = {
     activeClickAxisIndex: null,
     activeClickAxisDataKey: undefined,
   },
+  syncInteraction: {
+    active: false,
```

Comment:
Is this used?

Reply:
I think I'll need to use it once I integrate synchronisation and isActive prop on tooltip

Reply:
Gotcha